src/app/reviewer/[orgSlug]/study/[studyId]/review/study-results.tsx

```diff
@@ -39,7 +39,7 @@ export const StudyResults: FC<{
             <Stack>
                 <Group justify="space-between">
                     <Title order={4}>Study Results</Title>
-                    <JobReviewButtons job={job} decryptedResults={decryptedResults} />
+                    {decryptedResults && <JobReviewButtons job={job} decryptedResults={decryptedResults} />}
```

Comment:
I'd p refer to have `JobReviewButtons` return null if decryptedREsults aren't present vs doing the check here. I'll go ahead and merge and mixup in a followup tho